ts/features/itwallet/common/components/ItwUpcomingWalletBanner.tsx

```diff
@@ -1,32 +1,41 @@
 import React from "react";
-import { Banner } from "@pagopa/io-app-design-system";
-import { Linking } from "react-native";
+import { Banner, IOSpacer, VSpacer } from "@pagopa/io-app-design-system";
 import I18n from "../../../../i18n";
 import { useIOSelector } from "../../../../store/hooks";
 import { trialStatusSelector } from "../../../trialSystem/store/reducers";
 import { itwTrialId } from "../../../../config";
 import { SubscriptionStateEnum } from "../../../../../definitions/trial_system/SubscriptionState";
-import { isItwEnabledSelector } from "../../../../store/reducers/backendStatus";
 
-export const ItwUpcomingWalletBanner = () => {
-  const isItwEnabled = useIOSelector(isItwEnabledSelector);
+type ItwUpcomingWalletBannerProps = {
+  bottomSpacing?: IOSpacer;
+};
+
+export const ItwUpcomingWalletBanner = ({
+  bottomSpacing
+}: ItwUpcomingWalletBannerProps) => {
   const itwTrialStatus = useIOSelector(trialStatusSelector(itwTrialId));
 
-  if (!isItwEnabled || itwTrialStatus !== SubscriptionStateEnum.SUBSCRIBED) {
+  if (itwTrialStatus === SubscriptionStateEnum.ACTIVE) {
     return null;
   }
 
   return (
-    <Banner
-      color="neutral"
-      size="big"
-      pictogramName="notification"
-      title={I18n.t("features.itWallet.discovery.upcomingWalletBanner.title")}
-      content={I18n.t(
-        "features.itWallet.discovery.upcomingWalletBanner.content"
-      )}
-      action={I18n.t("features.itWallet.discovery.upcomingWalletBanner.action")}
-      onPress={() => Linking.openURL("https://io.italia.it/")} // TODO: [SIW-1716] Update the URL
-    />
+    <>
+      <Banner
+        testID="itwUpcomingWalletBannerTestID"
+        color="neutral"
+        size="big"
+        pictogramName="notification"
+        title={I18n.t("features.itWallet.discovery.upcomingWalletBanner.title")}
+        content={I18n.t(
+          "features.itWallet.discovery.upcomingWalletBanner.content"
+        )}
+        // action={I18n.t(
+        //   "features.itWallet.discovery.upcomingWalletBanner.action"
+        // )}
+        // onPress={() => Linking.openURL("https://io.italia.it/documenti-su-io/")} //At the moment, the page is not available
```

Comment:
Can we remove this commented code? The URL should be moved into the env but we already have a ticket to track this TODO. I'd restore the `// TODO: [SIW-1716] Update the URL` comment if that makes sense to you.

Reply:
If there is a specific ticket [SIW-1716], I would remove the commented code to make it cleaner and since I have already worked on this component I could take the ticket [SIW-1716]. What do you think?